bin/init-docker.sh

```diff
@@ -0,0 +1,27 @@
+echo "setting up docker container. This can take a few minutes depending on your netowrk speed"
+
+# clone the data repository
+git clone https://github.com/Scripta-Qumranica-Electronica/Data-files.git ./tmp-data
+
+# cd into the directory
+cd ./tmp-data
+
+# Cleanup up any images that might exist if running setup a second time
+docker stop SQE_Database && docker rm SQE_Database
+
+# Build the image
+docker build -t sqe-maria:latest .
+
+# start the container
+docker run --name SQE_Database -e MYSQL_ROOT_PASSWORD=none -d -p 3307:3306 sqe-maria:latest
```

Comment:
When I want to make changes to the database I load up a fresh instance of our container, make the changes, then run my backup script.  This backup script uses SELECT INTO, which writes queries out to a csv file.  It can only write to the local file system, so I add a volume for that "-v ~/sqe-mysql-backup:/tmp/backup" (we could just use ./db-backup or something like that for the local folder).  Can or should we add this into the install script here, or do you think it is too specialized?

Reply:
I guess my suggestion is to add -v ./db-backup:/tmp/backup to the docker run command.  That way I can run my database backup scripts on it and access the backup files it creates.  It is perhaps not vital right now, but it would be useful for Ingo and me.